pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotLeadControllerRestletResource.java

```diff
@@ -211,12 +212,16 @@ public List<String> getTableNames() {
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   public static class LeadControllerResponse {
+    @JsonProperty("isLeadControllerResourceEnabled")
```

Comment:
Can you verify if it should be `leadControllerResourceEnabled` or `isLeadControllerResourceEnabled`?
A JSON ser/de test will be very helpful to verify this and prevent breaking it in the future

Reply:
Added a test to verify this and refactored previously added ser/deser tests into a new util test class.